tf2_eigen/Doxyfile

```diff
@@ -0,0 +1,29 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "tf2_eigen"
+PROJECT_NUMBER         = master
+PROJECT_BRIEF          = "Eigen binding for tf2."
+
+INPUT                  = ./include ./mainpage.dox
+RECURSIVE              = YES
+OUTPUT_DIRECTORY       = doc_output
+
+EXTRACT_ALL            = YES
+SORT_MEMBER_DOCS       = NO
+
+GENERATE_LATEX         = NO
+
+ENABLE_PREPROCESSING   = YES
+MACRO_EXPANSION        = YES
+EXPAND_ONLY_PREDEF     = YES
+PREDEFINED             += TF2_PUBLIC=
+
+# Tag files that do not exist will produce a warning and cross-project linking will not work.
+TAGFILES += "../../../../doxygen_tag_files/cppreference-doxygen-web.tag.xml=http://en.cppreference.com/w/"
+# Consider changing "latest" to the version you want to reference (e.g. beta1 or 1.0.0)
+TAGFILES += "../../../../doxygen_tag_files/rcutils.tag=http://docs.ros2.org/latest/api/rcutils/"
```

Comment:
Since the package.xml only lists tf2 and tf2_ros as its dependencies, switch all these out with tf2/tf2_ros.